pytensor/link/jax/dispatch/loop.py

```diff
@@ -0,0 +1,52 @@
+import jax
+
+from pytensor.compile.mode import get_mode
+from pytensor.link.jax.dispatch.basic import jax_funcify
+from pytensor.loop.op import Scan
+from pytensor.tensor.type_other import NoneTypeT
+
+
+@jax_funcify.register(Scan)
+def jax_funcify_Scan(op, node, **kwargs):
+    # TODO: Rewrite as a while loop if only last states are used
+    if op.has_while_condition:
+        raise NotImplementedError(
+            "Scan ops with while condition cannot be transpiled JAX"
+        )
+
+    # Apply inner rewrites
+    # TODO: Not sure this is the right place to do this, should we have a rewrite that
+    #  explicitly triggers the optimization of the inner graphs of Scan?
+    update_fg = op.update_fg.clone()
+    rewriter = get_mode("JAX").optimizer
+    rewriter(update_fg)
```

Comment:
This issues the annoying Supervisor Feature missing warning... gotta clean that up